forms/htmleditor/TinyMCEConfig.php

```diff
@@ -416,31 +435,26 @@ protected function getEditorCSS() {
 	 * Generate gzipped TinyMCE configuration including plugins and languages.
 	 * This ends up "pre-loading" TinyMCE bundled with the required plugins
 	 * so that multiple HTTP requests on the client don't need to be made.
+	 *
+	 * @return string
 	 */
-	public function requireJS() {
-		require_once THIRDPARTY_PATH . '/tinymce/tiny_mce_gzip.php';
-
+	public function getScriptURL() {
+		// If gzip is disabled just return core script url
 		$useGzip = Config::inst()->get('HtmlEditorField', 'use_gzip');
-		$languages = array();
-
-		foreach(self::$configs as $configID => $config) {
```

Comment:
The current config might not belong to self::$configs, so this fix only loads its own language.